File: music-killer-java/music-killer-pojo/src/main/java/xyz/jdynb/entity/PlayInfo.java
package xyz.jdynb.entity;

import com.fasterxml.jackson.annotation.JsonAlias;
import io.swagger.v3.oas.annotations.media.Schema;
import lombok.experimental.Accessors;
import lombok.Data;

@Schema(description = "播放信息")
@Data
@Accessors(chain = true)
public class PlayInfo {

    @Schema(description = "码率", example = "128k")
    private Integer bitrate;

    @Schema(description = "时长", example = "1")
    private Integer duration;

    private String format;

    @Schema(description = "歌曲id", example = "1")
    @JsonAlias("rid")
    private Long musicId;

    private String sig;

    private String source;

    private Integer type;

    @Schema(description = "播放地址")
    private String url;

    private String user;

}

File: music-killer-android/app/src/main/java/xyz/jdynb/music/model/PlayInfo.kt
package xyz.jdynb.music.model


import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class PlayInfo(
    @SerialName("bitrate")
    var bitrate: Int = 0,
    @SerialName("duration")
    var duration: Int = 0,
    @SerialName("format")
    var format: String = "",
    @SerialName("musicId")
    var musicId: Long = 0,
    @SerialName("sig")
    var sig: String = "",
    @SerialName("source")
    var source: String = "",
    @SerialName("type")
    var type: Int = 0,
    @SerialName("url")
    var url: String = "",
    @SerialName("user")
    var user: String = ""
)